apps/gkyl_gyrokinetic_priv.h

```diff
@@ -152,20 +152,22 @@ struct gk_species_moment {
 struct gk_rad_drag {  
   int num_cross_collisions; // number of species we cross-collide with
   struct gk_species *collide_with[GKYL_MAX_SPECIES]; // pointers to cross-species we collide with
-  int collide_with_idx[GKYL_MAX_SPECIES]; // index of species we collide with
+  struct gk_neut_species *collide_with_neut[GKYL_MAX_SPECIES]; // pointers to neutral cross-species we collide with
+  int collide_with_idx[2*GKYL_MAX_SPECIES]; // index of species we collide with
```

Comment:
@jRoeltgen @akashukla 
why is MAX_SPECIES doubled in various places? If it's not high enough we should just change MAX_SPECIES in gkyl_util.h

Reply:
Since both gk_species and gk_neut_species can have GKYL_MAX_SPECIES, (most) radiation array maxes are set to 2*GKYL_MAX_SPECIES, since there is little difference in the structure of neutral and ion radiation. We may want to change the variable name or add a GKYL_MAX_NEUT_SPECIES or something, since currently GKYL_MAX_SPECIES is not actually the max number of species.